pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -548,8 +547,7 @@ func isNotFound(err error) bool {
 }
 
 func (os *OpenStack) Zones() (cloudprovider.Zones, bool) {
-	glog.V(1).Info("Claiming to support Zones")
-
+	glog.V(4).Info("Claiming to support Zones")
```

Comment:
ditto, although obviously this one is trivially enabled so doesn't communicate any new information to the log-viewer.